deployment/cdk/opensearch-service-migration/cdk.context.json

```diff
@@ -15,7 +15,7 @@
     },
     "sourceCluster": {
       "endpoint": "<SOURCE_CLUSTER_ENDPOINT>",
-      "version": "<SOURCE_CLUSTER_VERSION, ES_7_10 or OS_1_3>",
+      "version": "<SOURCE_CLUSTER_VERSION, ES_7.10 or OS_1_3>",
```

Comment:
We have some parsing in here when we deploy a source cluster that requires the `.` syntax